### ISSUE TITLE
Cell actions not accessible with VoiceOver

### ISSUE THREAD

**Issue by BasThomas** [GitHawkApp/GitHawk#1881]
<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.22.0 (1529261281)
Device: iPhone X (iOS 11.4)
TestFlight: true
</pre>
</details>
